### DISCUSSION TITLE
Make 'Chart Type' control accessible by using keyboard.

### DISCUSSION THREAD

**Discussion by msftedad** [apache/superset#33071] [Ideas]
'Chart Type' control is not accessible by using keyboard.

#### How to reproduce the bug
1. Open URL: https://titanweb-testenv/superset/dashboard/268/ in Edge browser.
2. Navigate to the + icon appearing in header and activate charts control.
3. Create new chart page appears. Select any chart and activate create new chart control.
4. New page appears. Navigate to the chart type dropdown
5. Navigate to 'Chart type dropdown by using tab key.
6. Observe that 'Chart Type' control is not accessible by using keyboard.
### Expected results
Control(s) must be accessible to keyboards and other assistive technology. Common causes of this problem include a) the element does not have a proper role assigned, b) the element needs tab-index="0" attribute to be focusable, or c) the component is not registering keypresses.
what you expected to happen.

### Actual results
'Chart Type' control is not accessible by using keyboard.

### Environment
Platform: Web
OS: Windows_11
Version: 22H2
OS Build: 25174.1010
Browser: New edge (105.0.1343.50) 
URL:  https://titanweb-testenv/superset/dashboard/268/ 
(please complete the following information):

### Additional context
1. Same issue repro for controls all the under 'Data' tab and 'Customize tab'.
2. Same issue repro for information (i) controls under 'Data' tab and 'Customize tab'.
3. Issue is repro for similar controls throughout the page.

Attachments:
![Note section bug](https://github.com/apache/superset/assets/93735775/6fa9d7a0-55f2-436f-8a40-d9f038f4b212)

https://github.com/apache/superset/assets/93735775/fcc95f06-a877-4925-8317-761288a4212f

**Comment by rusackas** [comment]
I seem to be following you around, so sorry for the somewhat redundant comments. I really do appreciate that you filed these as separate issues, however, rather than a "wall of problems" ticket :)

Again, we might be forced to close this as stale and non-bug unless anyone wants to contribute a PR

**Comment by rusackas** [comment]
Anyone object to closing this? It'd stay on the Accessibility project board in case anyone wants to volunteer to pick it up.

**Comment by rusackas** [comment]
Moving this to an "Ideas" thread to keep the backlog more actionable. I hope the effort (and related accessibility efforts) find a volunteer.
